src/views/scans/showScansModal.tsx

```diff
@@ -18,12 +18,22 @@ import { type Scan, type ScanJobType } from '../../types/types';
 interface ShowScansModalProps {
   isOpen: boolean;
   scan?: Pick<Scan, 'name'>;
-  scanJobs?: Pick<ScanJobType, 'id' | 'end_time' | 'report_id' | 'status'>[];
+  scanJobs?: (Pick<ScanJobType, 'id' | 'start_time' | 'report_id' | 'status'> & { end_time: Date | null })[];
```

Comment:
required: `end_time` is already included in `ScanJobType`. if there is the potential that `end_time` doesn't exist it should be made optional within the the `ScanJobType` under `types.ts`

Change `ScanJobType` under `types.ts`
```
...
  report_id: number;
  start_time: Date;
  end_time?: Date;
  systems_count: number;
...
```

Change this current line towards
```
scanJobs?: Pick<ScanJobType, 'id' | 'start_time' | 'report_id' | 'status' | 'end_time'>[];
```